.github/workflows/cd-syft.yml

```diff
@@ -335,7 +335,7 @@ jobs:
 
       - name: Copy helm repo files from Syft Repo
         run: |
-          rm -rf ghpages/helm/repo/
+          rm -rf ghpages/helm/
```

Comment:
maybe we could 
`mkdir -p ghpages/helm`

Reply:
Updated the remove command to this:
 rm -rf ghpages/helm/*

This doesn't delete the directory, only the contents of the directory